src/base/preferences.cpp

```diff
@@ -448,6 +448,26 @@ void Preferences::setActionOnDblClOnTorrentFn(int act)
     setValue("Preferences/Downloads/DblClOnTorFn", act);
 }
 
+bool Preferences::deleteAddedTorrents() const
```

Comment:
Well, actually it's either bad design or these properties relate to the application as a whole (i.e. must belong to the Application class).